src/api/Publisher/getPaymentMethods.ts

```diff
@@ -1,7 +1,16 @@
 import fetchWithJWT from 'util/fetchHelper';
 import getApiURL from 'util/environmentHelper';
 
-const getPaymentMethods = () => {
+const getPaymentMethods = (): Promise<{
+  responseData: {
+    paymentMethods: {
+      methodName: string;
+      id: string;
+      [key: string]: unknown;
```

Comment:
I'm curious, why did you add this line this way? `[key: string]: unknown;` 
In paymentMethods we have also logoUrl and paymentGateway, nothing more